### DISCUSSION TITLE
Can't pair Moes ZTS-EUR-C curtain switch

### DISCUSSION THREAD

**Discussion by Pazystamas** [Koenkk/zigbee2mqtt#18290] [Devices]
Hi,
I have [Moes curtain switch](https://www.zigbee2mqtt.io/devices/ZTS-EUR-C.html ) and before migrating to Z2M it worked fine with ZHA. I migrated all devices successfully, except this switch. According to manual you need to "press the switch button for 6 times, hold on the 6th time and then release", I tried with all 3 buttons, it flashes, but Z2M cant find device and nothing shows up on logs. 
How to pair it?

**Comment by Pazystamas** [comment]
Finally paired successful, but had to change zigbee channel from 11 to 15. Channel 25 also did not work (at least for me). Also had to reverse motor setting to show correctly on home assistant.

**Comment by denquer** [comment]
Hi,
I have the same issue but switching the channel didn't work for me.

I've tried the following channels: 11, 15, 20, 25. Anyone else who solved the issue or with a good idea?
@Pazystamas Which version of z2m have you been using?

Since the device is meant to be supported by z2m I'm quite surprised.

> Specs:
> Zigbee2MQTT version
>     [2.5.1](https://github.com/Koenkk/zigbee2mqtt/releases/tag/2.5.1) commit: [unknown](https://github.com/Koenkk/zigbee2mqtt/commit/unknown)
> 
> Coordinator type
>     EmberZNet
> 
> Coordinator revision
>     7.4.3 [GA]
> 
> Frontend version
>     0.9.16
> 
> zigbee-herdsman-converters version
>     23.69.0
> 
> zigbee-herdsman version
>     4.3.1

**Reply by Pazystamas** [reply]
Hi,
Same z2m version, but I'm using ZStack3x0 Sonoff coordinator.
Maybe it is too far? I had no problems after pairing in those two years.

**Reply by denquer** [reply]
@Pazystamas 
Embarassing but true... I must admit that I didn't think of this option, so thanks for stating the obvious.

The curtain switch seems to have a very low signal range. Placing a router (IKEA INSPELNING) right besides it solved the issue.
